openwisp_monitoring/monitoring/tests/test_monitoring_notifications.py

```diff
@@ -127,10 +128,12 @@ def test_general_check_threshold_crossed_for_long_time(self):
             self.assertEqual(m.is_healthy_tolerant, True)
             self.assertEqual(Notification.objects.count(), 0)
 
+        time.sleep(0.1)
```

Comment:
make sure this sleep is executed only when in UDP mode and use the function mentioned above